include/benchmark/reporter.h

```diff
@@ -155,7 +156,7 @@ class BenchmarkReporter {
 // Simple reporter that outputs benchmark data to the console. This is the
 // default reporter used by RunSpecifiedBenchmarks().
 class ConsoleReporter : public BenchmarkReporter {
- public:
+public:
```

Comment:
yes. clang-format with google style setting. whatever that says is what we go with :)